_sass/_base.scss

```diff
@@ -208,3 +209,235 @@ pre {
+     * Figures
+     */
+    figure > img {
+        display: block;
+    }
+    
+    figcaption {
+        font-size: $small-font-size;
+    }
+    
+    
+    
+    /**
+     * Lists
+     */
+    ul, ol {
+        margin-left: $spacing-unit;
+    }
+    
+    li {
+        > ul,
+        > ol {
+             margin-bottom: 0;
+        }
+    }
+    
+    
+    
+    /**
+     * Headings
+     */
+    h1, h2, h3, h4, h5, h6 {
+        font-weight: 300;
+        color: #fff;
+    }
+    
+    
+    
+    /**
+     * Links
+     */
+    a {
+        color: #0096cf;
+        text-decoration: none;
+    
+        &:visited {
+            color: #005d81;
+        }
+    
+        &:hover {
+            color: #0096cf;
+            text-decoration: underline;
+        }
+    }
+    
+    
+    
+    /**
+     * Blockquotes
+     */
+    blockquote {
+        color: inherit;
+        border-left: 4px solid #5c6773;
+        padding-left: $spacing-unit / 2;
+        font-size: 18px;
+        letter-spacing: -1px;
+        font-style: italic;
+    
+        > :last-child {
+            margin-bottom: 0;
+        }
+    }
+    
+    
+    
+    /**
+     * Code formatting
+     */
+    pre,
+    code {
+        font-size: 15px;
+        border: none;
+        border-radius: 3px;
+        background-color: #191f26;
+    
+    }
+    
+    code {
+        padding: 0 0.2em;
+    }
+    
+    pre {
+        padding: 8px 12px;
+        overflow-x: auto;
+    
+        > code {
+            border: 0;
+            padding-right: 0;
+            padding-left: 0;
+        }
+    }
+    
+    
+    
+    /**
+     * Wrapper
+     */
+    .wrapper {
+        max-width: -webkit-calc(800px - (#{$spacing-unit} * 2));
+        max-width:         calc(800px - (#{$spacing-unit} * 2));
+        margin-right: auto;
+        margin-left: auto;
+        padding-right: $spacing-unit;
+        padding-left: $spacing-unit;
+        @extend %clearfix;
+    
+        @include media-query($on-laptop) {
+            max-width: -webkit-calc(800px - (#{$spacing-unit}));
+            max-width:         calc(800px - (#{$spacing-unit}));
+            padding-right: $spacing-unit / 2;
+            padding-left: $spacing-unit / 2;
+        }
+    }
+    
+    
+    
+    /**
+     * Clearfix
+     */
+    %clearfix {
+    
+        &:after {
+            content: "";
+            display: table;
+            clear: both;
+        }
+    }
+    
+    
+    
+    /**
+     * Icons
+     */
+    .icon {
+    
+        > svg {
+            display: inline-block;
+            width: 16px;
+            height: 16px;
+            vertical-align: middle;
+    
+            path {
+                fill: $grey-color;
+            }
+        }
+    }
+    
+    /**
+     * Text styling
+     */
+    strong, b {
+      font-weight: bold;
+    }
+    
+    /**
+    * Code higlighting
+    */
+    .highlight {
+        background-color: #191f26 !important;
+        border: none !important;
+    }
+}
```

Comment:
Nit: Can we add a newline here?

Reply:
Haha yeah sorrry about that it was copied from somewhere else and I didn’t change that but, will sort out